src/pages/home/home.ts

```diff
@@ -96,6 +96,9 @@ export class HomePage {
           localStorage.setItem("defaultTokens", JSON.stringify(this.tokens));
 
           await this.loadTokens();
+
```

Comment:
below here in case user add a token, we should add a listener for the new added token also